kuzzleio/documentation
# fix fix-indent package remove breaklines

## What does this PR do?
We are using [fix-indents](https://www.npmjs.com/package/fix-indents) package to indent our snippets but it remove the breaklines and decrease the visibility of the snippets.

Remove lines 39-41 let fix-indent keep breaklines.

### How should this be manually tested?

  - Step 1 : npm install
  - Step 2 : npm run repositories clone
  - Step 3 : npm run dev
  - Step 4 : Check snippets 

### Other changes
/

### Boyscout
/
